seqr/views/apis/variant_search_api_tests.py

```diff
@@ -439,16 +439,25 @@ def _get_variants(results_model, **kwargs):
         }))
         self.assertEqual(response.status_code, 200)
         response_json = response.json()
-        self.assertDictEqual(response_json, {
+        empty_search_response = {
             'searchedVariants': [], 'search': {
                 'search': SEARCH,
                 'projectFamilies': [],
                 'totalResults': 0,
-        }})
+        }}
+        self.assertDictEqual(response_json, empty_search_response)
         results_model = VariantSearchResults.objects.get(search_hash=SEARCH_HASH)
         mock_get_variants.assert_called_with(results_model, sort='xpos', page=1, num_results=100, skip_genotype_filter=True, user=self.no_access_user)
 
-        results_model.delete()
+        VariantSearchResults.objects.filter(search_hash=SEARCH_HASH).delete()
```

Comment:
it should, but doesn't for some reason. Since manipulating models like this is never done in the app I don't think its worth the time to debug why that way of deleting the model doesn't work and doing this change is fine